wpilibc/src/main/native/include/frc/TimedRobot.h

```diff
@@ -35,14 +37,22 @@ class TimedRobot : public IterativeRobotBase, public ErrorBase {
   /**
    * Get the time period between calls to Periodic() functions.
    */
-  double GetPeriod() const;
+  units::second_t GetPeriod() const;
```

Comment:
Note this is a breaking change, but I think it's likely rarely used / easy to fix if it is used, so I'm okay with it.